apps/web/public/static/locales/fr/common.json

```diff
@@ -68,7 +68,7 @@
   "password_reset_instructions": "Si vous n'en avez pas fait la demande, vous pouvez ignorer cet e-mail en toute sécurité et votre mot de passe ne sera pas modifié.",
   "event_awaiting_approval_subject": "En attente de validation : {{title}} à {{date}}",
   "event_still_awaiting_approval": "Un événement est toujours en attente de votre validation",
-  "booking_submitted_subject": "Réservation confirmée : {{eventType}} avec {{name}} le {{date}}",
```

Comment:
I think we should be able to write a utility that can ensure that a string has the same variables in all languages. I have seen this kind of language-specific issue many times. It's difficult to identify during testing and only the users who use the language end up facing the issue.